webpack.dev.js

```diff
@@ -106,12 +106,30 @@ module.exports = {
     historyApiFallback: {
       disableDotRule: true,
     },
-    proxy: {
-      "/api": {
+    proxy: [
+      {
+        context: ["/api"],
         target: API_URL,
         pathRewrite: { "^/api": "" },
         logLevel: "debug",
+        bypass: (req, res, _proxyOptions) => {
+          if (req.url.endsWith("/diff")) {
+            res.send(branchDiffJson);
+          }
+
+          if (req.url.includes("/diff/types")) {
+            res.send(definitionTypeDiffJson);
+          }
+          /*
+          if (req.url.includes("/diff/terms")) {
+            res.send(definitionDiffJson);
+          }
+          if (req.url.includes("/definitions/by-name")) {
+            res.send(definitionsByNameJson);
+          }
+          */
+        },
```

Comment:
We can leave the `bypass` field out I think. Those include stubs specific to the Share API.